### DISCUSSION TITLE
2023-06-21 Weekly Meeting 112 Notes/Agenda

### DISCUSSION THREAD

**Discussion by funksoup** [omigroup/omigroup#398] [General]
Weekly Meeting 112 is happening on Wednesday, June 21, 2023 from 9:00 AM PST (16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- MSF new membership process:
   - [New membership form](https://metaverse-standards.org/members/membership-application-form/)
   - [Register as POG (Pre-qualified Organization)](https://docs.google.com/forms/d/e/1FAIpQLScW4ks9vORd-ROx0248hcA9d8-Vn28yfJMCKFedtjxHS4k-ww/viewform)?
   - https://github.com/omigroup/omigroup/issues/399

- #362 

### UPDATES:

- MSF delegates
- Media group
- Avatar interop group
- GLTF group
- World Hops

**Comment by madjin** [comment]
I've been trying to coordinate atleast one MSF meetup event to field trip here in OMI, because it's about a constellation of communities helping build an open metaverse and we should see each other as neighbors. Been discussing with [avatar-wearables](https://github.com/MetaverseStandards/Avatar-Wearables) group chairs if we can coordinate a meetup together, atleast one during a planned bi-weekly:

- We've tried to share / promote meetups in between MSF bi-weekly, didn't work
- Scheduling a field trip during *one* from MSF -> OMI can help build trust / ease anxiety for first timers

Maybe we can discuss a plug fest / hackathon / field trip events / open metaverse conf together? M3 would host [field trips](https://twitter.com/dankvr/status/1558953913130401792), many during first year of formation... and it seems like so many of the newer open metaverse communities are missing out on these important cultural experiences. The metaverse is bigger than one org, I want different metaverse orgs to cross-over and not work within a vacuum and to check out what others are working on - as if we're all on the same campus / city.

**Reply by mrmetaverse** [reply]
Maybe we donate towards a hackathon to implement ours/others protocols. Let's help others actually build things with these protocols. Specifically, we could make this about wearables interop. 

3 day jam, plug-fest with prizes. 

cc: #362

**Reply by mrmetaverse** [reply]
maybe companies can sponsor the best implementations of their APIs. i.e. "build something in this hackathon using our API and the best creation gets a Quest II"

**Reply by mrmetaverse** [reply]
We want to avoid "clever tricks" We should strive for the spec to be as clean as possible. 
Our spec design should not surprise people, it should match their intuitive expectations. 

<img width="957" alt="Screenshot 2023-06-21 at 12 45 59 PM" src="https://github.com/omigroup/omigroup/assets/63426722/f3e6f507-b460-4ec3-b749-ce338c472959">

**Comment by mrmetaverse** [comment]
about OMI_personality https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_personality

**Comment by mrmetaverse** [comment]
Demo signup

**Reply by mrmetaverse** [reply]
- Jimmy6DOF

**Comment by mrmetaverse** [comment]
# MSF new sign-up process 
- TO DO: https://github.com/omigroup/omigroup/issues/399
- new bylaws -> new membership process 
- we have to kinda officially join 
- <img width="1134" alt="Screenshot 2023-06-21 at 12 20 56 PM" src="https://github.com/omigroup/omigroup/assets/63426722/2f0f1e40-2eb9-440c-8ca2-f7a98ca04463">
- similar questions to the Open Collective updated filing #361 
- We are filing as a POG, something our MSF Delegates worked hard to manifest as an option for groups like ours (a community organization that works in open source, or non-profit, etc. that creates standards) Our community made a big impact driving this forward.
- 
<img width="1113" alt="Screenshot 2023-06-21 at 12 21 54 PM" src="https://github.com/omigroup/omigroup/assets/63426722/d2da8f64-bc62-4800-8033-61a74aa2b840">
- the deadline for board nomination and elections is coming up soon. We want to get our filing in very soon to ensure that someone from our community can have a voice on the board. 
- there is another form though we are not sure if we have to fill this out too. MIght be old. 
- <img width="881" alt="Screenshot 2023-06-21 at 12 23 57 PM" src="https://github.com/omigroup/omigroup/assets/63426722/eb50b3cb-8346-4c25-ba68-285c4036d373">

**Comment by madjin** [comment]
Here's a markdown table of the worlds I've built to help illustrate our neighboring communities building the open metaverse. I want to try to inspire orgs to prevent them from thinking its them vs the world. Even if we don't meet up in these worlds, perhaps we can atleast use them for creating art for a section in the [zine](https://hackmd.io/@XR/m3-zine/) that M3 is working on?

| Project | Link | Image |
| --- | --- | --- |
| FrameVR | [https://framevr.io/omi](https://framevr.io/omi) | ![image](https://github.com/omigroup/omigroup/assets/32600939/23d56dad-8b97-4f1d-b80b-ad00c9925384) |
| Hyperfy Huts | [https://hyperfy.io/msf](https://hyperfy.io/msf) | ![image](https://github.com/omigroup/omigroup/assets/32600939/21741958-0798-4f94-9ad5-c2c495cfe869) |
| Hyperfy Dawn | [https://hyperfy.io/dawn](https://hyperfy.io/dawn) | ![image](https://github.com/omigroup/omigroup/assets/32600939/134fda82-06ff-4bfc-a46b-1cb848268ba2) |
| Hyperfy Hacklab | [https://hyperfy.io/hacklab](https://hyperfy.io/hacklab) | ![image](https://github.com/omigroup/omigroup/assets/32600939/161d3c36-6919-4eea-a7b4-dc4a982126e2) |
| Hyperfy Neon Street | [https://hyperfy.io/neonstreet/~piRH](https://hyperfy.io/neonstreet/~piRH) | ![image](https://github.com/omigroup/omigroup/assets/32600939/bf41cc73-2fd1-43c6-865d-ae6d3d686c4f) |

**Comment by mrmetaverse** [comment]
# Help students with a impactful webxr project
Funksoup is working with Students in Ethiopia to create an avatar role-playing game. 
They see this as an environment to jump into different Ethiopian ethnicities and culture. 
The project has been approved for funding, and she is seeking help and feedback. Would like to posit this as a call for participation. 

This will become a call-for-participation. 
<img width="843" alt="Screenshot 2023-06-21 at 12 29 31 PM" src="https://github.com/omigroup/omigroup/assets/63426722/eb0d895e-5c72-412d-8211-373890905c11">

**Reply by mrmetaverse** [reply]
<img width="1035" alt="Screenshot 2023-06-21 at 12 30 02 PM" src="https://github.com/omigroup/omigroup/assets/63426722/1ba44d0e-2fa1-4f61-acf8-a7e54edc2c88">

**Comment by mrmetaverse** [comment]
# Mapping the Metaverse update 
- Jimmy6DOF and Keyframe had a couple of sessions since our discussion two weeks ago. 
- discussed the effort Keyframe has been compiling for the AWE presentation he gave, and its intent to categorize the metaverse along a system of world types, visual accesses, etc.  https://keyframe-entertainment.com/miw/
- Dr. Kim Nevelsteen is working on a process to link your internal file system and external file systems (IPSME!). Like an operating system that functions locally and can create links between various other metaverse platforms. Allowing bridges between even proprietary systems. 
- discussions highlighted that there are a lot of opinions around the topic of mapping and connecting the metaverse. 
- Jimmy6DOF wants to create a free service as a data warehouse that houses a bunch of records associated with vetted Metaverse properties of sorts, that can create a set of information that can be extracted and used by people like Keyframe to use the data, apply his references, to tell the data story he wants to tell using that data. J6D wants an unopinionated account of the metaverse that can be built on top of. 
- a lot of work has been done before, an airtable, a website setup, etc. A lot of activity was put in before, and J6D believes it is still a worthy endeavour to serve as a storage zone of sorts for others to use to try to map the metaverse in their way.
